dotcom-rendering/src/web/components/Kicker.tsx

```diff
@@ -29,7 +31,30 @@ export const Kicker = ({
 	color,
 	showPulsingDot,
 	showSlash = true,
+	showLineBreak = true,
 }: Props) => {
+	const [removeKickerSlash, setRemoveKickerSlash] = useState(false);
+
+	useEffect(() => {
+		setRemoveKickerSlash(
+			// eslint-disable-next-line @typescript-eslint/no-unnecessary-condition -- window not available on server
+			window &&
+				window.guardian.config.tests.removeKickerSlashesVariant ===
+					'variant',
+		);
+	}, []);
+
+	if (removeKickerSlash) {
+		return (
```

Comment:
Once this is moved out from behind the 0% test, we can remove the 'removeKickerSlash' check, and just replace the original return value for this component (i.e. the one starting on ln.59) with the return value of this branch (i.e. ln.49-55).